web/src/components/vis/AnovaTableTile.vue

```diff
@@ -89,7 +89,8 @@ vis-tile-large(v-if="plot", title="Anova Table", :loading="plot.loading", expand
     v-btn(flat, dark, block, @click="downloadTable")
       v-icon.mr-2 {{ $vuetify.icons.save }}
       | Download Table
-  anova-table.horizontalScrollable(v-if="plot.data", :data="tableData", :threshold="threshold", v-model="selected")
+  anova-table.horizontalScrollable(v-if="plot.data", :data="tableData",
+      :threshold="threshold", v-model="selected")
```

Comment:
When we break a long Pug line, we tend to go all the way: put each attribute on a single line. (But I can't remember if we are supposed to retain the trailing commas in this situation--@subdavis?)

So something like:
```pug
elem(
    v-if="...",
    :data="...",
    .
    .
    .
)
```